assets/styles/layout/_layout.scss

```
@@ -36,7 +36,7 @@ body {
     @include media-breakpoint-up(lg) {
       .subs-show & {
         grid-template-areas: 'subs main sidebar';
-        grid-template-columns: 1fr 3fr 1fr;
+        grid-template-columns: 1fr 6fr 2fr;
```

So I think that will become this again:

```suggestion
       grid-template-columns: 1fr 3fr 1fr;
```

Or we could even for go: `300px auto 300px;`